package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "clark",
   "displayName": "CLARK: Cybersecurity Labs and Resource Knowledge-base",
-  "version": "4.4.1",
+  "version": "4.5.0",
```

Comment:
Shouldn’t this be version: 4.5.2?

Reply:
We're reverting back to the 4.5 and thats 4.4.1 for the last release.

Reply:
ohh right, i read the previous version wrong, thought it said 4.5.1. All good